model/invoice.rb

```diff
@@ -73,6 +73,8 @@ def charge
         "payment_intent" => payment_intent.id
       })
       save(columns: [:status, :content])
+      project.update(reputation: "verified") if amount > 5
```

Comment:
Why did we choose $5 as threshold?

Reply:
It was an arbitrary choice. $5 seemed like a good enough value. I'm open to different amounts as well.